fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -1088,7 +1099,12 @@ protected void unprotectedCommitTransaction2PC(TransactionState transactionState
             return;
         }
         // update transaction state version
-        transactionState.setCommitTime(System.currentTimeMillis());
+        long commitTime = System.currentTimeMillis();
+        if (preCommitTime == commitTime || preCommitTime == 0) {
+            commitTime += 1;
+        }
+        preCommitTime = commitTime;
+        transactionState.setCommitTime(commitTime);
```

Comment:
why add preCommitTime logic?

Reply:
Multiple transactions commit time may be consistent, this logic is to ensure that the transaction commit time is strictly incremental, to prevent handler thread stuck